tests/Helpers/Models/MethodConfigTest.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Moves\TestHelpers\Models;
```

Comment:
As I mentioned in slack, the reason you're having trouble accessing the classes is that the namespace is wrong. It should be `Tests\Helpers\Models`, to match the directory names.